data-raw/DJFMP.R

```diff
@@ -77,23 +126,26 @@ DJFMP <-  bind_rows(
            is.infinite(Count) ~ "Unknown length",
            is.na(Length)~ "No fish caught",
            TRUE ~ NA_character_), # Add reasoning for an NA lengths (all "No Fish Caught" for FMWT)
-         Count=if_else(is.infinite(Count), Total, Count))%>%
-  filter(Length!=0 | is.na(Length))%>%
+         Count=ifelse(is.infinite(Count), Total, Count))%>%
+  dplyr::filter(Length!=0 | is.na(Length))%>%
   select(-Total, -TotalMeasured, -Group)%>%
-  left_join(DJFMP_stations, by = "Station") %>%
+  left_join(DJFMP_stations, by = c("Station"="StationCode")) %>%
   # Add species names
-  left_join(Species %>%
-              select(USFWS_Code, Taxa) %>%
-              filter(!is.na(USFWS_Code)),
+  left_join(USFWS_Species,
```

Comment:
why is this used instead of our unified species table?

Reply:
USFWS_Species is just a subset of the Species table apparently. Seems as though it was done to limit the species codes to USFWS only and not introduce lines of NAs when joining, i.e., species codes from other surveys that DJFMP doesn't use.

Left as is for now.